Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,7 +132,16 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
+            if ((ValidateArrayForManipulation(array)) && (ConfirmThatTargetIndexToManipulateIsWithinTheRangeOfTheArray(array, "Remove", index)))
+            {
+                var shortenedArray = MakeAShorterArray(array);
+
+                for (int i = 0; i < shortenedArray.Length; i++)
+                {
+                shortenedArray[i] = i >= index ? array[i + 1] : array[i];
```

Comment:
Keep the formatting the same throughout your document. :)
Good use of ?: but in my opinion this could be more readable by placing ( ) around the i >= index.